terraform/deployments/concourse-iam/main.tf

```diff
@@ -63,7 +63,7 @@ resource "aws_iam_role" "govuk_concourse_terraform_planner" {
         "Effect" : "Allow",
         "Action" : "sts:AssumeRole",
         "Principal" : {
-          "AWS" : "arn:aws:iam::047969882937:role/cd-govuk-ci-concourse-worker"
+          "AWS" : "arn:aws:iam::${var.concourse_aws_account_id}:role/cd-govuk-ci-concourse-worker"
```

Comment:
🤔  I don't think the Concourse CI team need to `terraform plan` against integration. Should we extract this into a separate deployment, or perhaps have a `count = 0 if env != test` on this resource?

Reply:
good catch, I need to put a count=0 for that if the env is not test